library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -1299,73 +1331,74 @@ def GetDevTypeDescription(type, longdescr)
     # @return dumped settings (later acceptable by Import())
     def Export(devregex)
       devs = Filter(@Devices, devregex)
-      Builtins.y2debug("Devs=%1", devs)
-      Convert.convert(devs, from: "map", to: "map <string, map>")
+      log.debug("Devs=#{devs}")
+      devs
     end
 
     # Were the devices changed?
     # @return true if modified
     def Modified(devregex)
       devs = Filter(@Devices, devregex)
       original_devs = Filter(@OriginalDevices, devregex)
-      Builtins.y2debug("OriginalDevs=%1", original_devs)
-      Builtins.y2debug("Devs=%1", devs)
+      log.debug("OriginalDevs=#{original_devs}")
+      log.debug("Devs=#{devs}")
       devs == original_devs
     end
 
+    # It returns an array with the next N devices available
+    # for the given device type.
```

Comment:
ehm. No. It returns something like an array of N free indexes. Device name has to be constructed by caller. And it is silently expected that it will be ```<type><i>```

Honestly I dislike this function and it is candidate for removal for me.